columnflow/tasks/plotting.py

```diff
@@ -210,6 +210,14 @@ def create_branch_map(self):
             for cat_name in sorted(self.categories)
         ]
 
+    def workflow_requires(self):
+        reqs = super().workflow_requires()
+
+        # no need to require merged histograms since each branch already requires them as a workflow
+        reqs.pop("merged_hists", None)
```

Comment:
are you sure we can do this? I'd assume this means that when running plotting tasks via job submission, job dependencies will only be resolved after having already submitted the corresponding branch (meaning that if requirements are not already met, the whole workflow will run in each submitted job)

Reply:
We can also change this to only have an effect for local workflows.